### ISSUE TITLE
Error when running in Mac OS 10.10

### ISSUE THREAD

**Issue by zencoding** [teradeep/demo-apps#1]
I get the below error when running, 

Error /usr/local/share/lua/5.1/nn/Sequential.lua:37: calling 'updateOutput' on bad self (dW must == 1)

Looks like some parameter value is incorrect in one of the layers, since I don't have the model parameters, I could not find out the exact issue.

**Comment by vgokhale** [comment]
Have you updated Torch7 recently? I believe support for stride > 1 was added to convMM in commit 0298277b5e2478668bf82c0cef7b401139f9dc89. Could you try updating Torch (if you haven't recently) and try again?

I'm a bit puzzled by your statement regarding the model parameters. The network should be available to you since you are using it in the demo.

**Comment by zencoding** [comment]
Thanks, that made it work.  

BTW, what I meant by parameters is the arguments that set in the layer for example SpatialConvolutionMM. When I try to print the model, I get the below but I was trying to get the value of filter size and featuremaps that were set for SpatialConvolutionMM.  I guess my question would be how can I extend/enhance this model with more layers or training data?

nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> (14) -> (15) -> (16) -> (17) -> (18) -> (19) -> (20) -> (21) -> (22) -> (23) -> output]
  (1): nn.SpatialConvolutionMM
  (2): nn.ReLU
  (3): nn.SpatialConvolutionMM
  (4): nn.SpatialMaxPooling
  (5): nn.ReLU
  (6): nn.SpatialConvolutionMM
  (7): nn.SpatialMaxPooling
  (8): nn.ReLU
  (9): nn.SpatialConvolutionMM
  (10): nn.SpatialMaxPooling
  (11): nn.ReLU
  (12): nn.SpatialConvolutionMM
  (13): nn.ReLU
  (14): nn.View
  (15): nn.Dropout
  (16): nn.Linear(2304 -> 4096)
  (17): nn.ReLU
  (18): nn.Dropout
  (19): nn.Linear(4096 -> 4096)
  (20): nn.ReLU
  (21): nn.Dropout
  (22): nn.Linear(4096 -> 1000)
  (23): nn.SoftMax
}

**Comment by vgokhale** [comment]
You can do 

    model = torch.load("model.net")
    {model}

or 

    {model.modules[i]}

where i can be any value between 1 and (for this network) 23.
